src/pages/[platform]/build-a-backend/auth/connect-your-frontend/manage-user-sessions/index.mdx

```diff
@@ -0,0 +1,88 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Manage user sessions',
+  description: 'Learn how to manage user sessions',
+  platforms: [
+    // 'android',
```

Comment:
platform list might need a cleanup here

Reply:
filed an issue for fast follow. need to port in mobile content for this page